src/contexts/cart-context.js

```diff
@@ -0,0 +1,17 @@
+import { createContext, useContext, useReducer } from "react";
+import { cartReducer } from "../reducers/cart-reducer";
```

Comment:
Suggestion: can use barrel export

so that you can directly import from `../reducers`